en/docs/learn/integration-tutorials/sending-a-simple-message-to-a-service.md

```diff
@@ -266,11 +254,11 @@ You can now configure the mediation logic to handle requests.
 
     Now let's add a **Respond** mediator at the end of the in sequence to send the response message from the healthcare service back to the client.
 
-8. Click on the **+** icon in the sequence to add a Respond mediator after the Call mediator.
+8. Click on the **+** icon in the sequence to add a **Respond** mediator after the Call mediator.
 
     <a href="{{base_path}}/assets/img/learn/tutorials/sending-simple-message-to-service/add-respond.png"><img src="{{base_path}}/assets/img/learn/tutorials/sending-simple-message-to-service/add-respond.png" alt="add respond" width="80%"></a>
 
-9. From the **Palette**, select **Respond** mediator under the **Mediators** > **Generic** section.
+9. From the **Palette**, select **Respond** mediator under the **Mediators** > **Favourites** section.
```

Comment:
For me, the Respond mediator is not listed under the favourites section. shall we recheck this?
<img width="404" alt="respond-mediator" src="https://github.com/user-attachments/assets/a5254a3f-2368-491d-922d-a1c7c267432f" />

Reply:
The respond mediator is listed under the favourite section. Were you able to check with the latest vscode plugins?

Reply:
Removed in [f6ebcad](https://github.com/wso2/docs-mi/pull/1338/commits/f6ebcad1c6d7aa646ecede14f86a733976e15401)